cmd/single/start.go

```diff
@@ -71,9 +72,11 @@ func startAdmin(ctx context.Context, cfg Admin) error {
 	}
 
 	logger.Infof(ctx, "Seeding default projects...")
-	projects := []string{"flytesnacks"}
-	if len(cfg.SeedProjects) != 0 {
-		projects = cfg.SeedProjects
+	projects := []config.Project{{Name: "flytesnacks", Description: "flytesnacks project"}}
+	if len(cfg.SeedProjects) > 0 {
+		for _, project := range cfg.SeedProjects {
+			projects = append(projects, config.Project{Name: project.Name, Description: project.Description})
+		}
```

Comment:
AFAIK doing it this way would keep project `flytesnacks` while it was dropped in favor of configured projects previoulsy